index.bs

```diff
@@ -461,8 +461,8 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
           dictionary MediaCapabilitiesKeySystemConfiguration {
             required DOMString keySystem;
             DOMString initDataType = "";
```

Comment:
The problem is different: for audio and video robustness, we have logic that says that if you set one or the other, there must be a configuration. Issue is that if you create a `MediaCapabilitiesKeySystemConfiguration`, per spec, it will set these two properties and always fail the check in https://w3c.github.io/media-capabilities/#mediaconfiguration

Reply:
I think this is resolved by https://github.com/w3c/media-capabilities/pull/138 (the nested audio/video =dictionary is optional and non-defaulted)